content/en/docs/reference/node/systemd-watchdog.md

```diff
@@ -0,0 +1,65 @@
+---
+content_type: "reference"
+title: Kubelet Systemd Watchdog
+weight: 80
+---
+
+{{< feature-state feature_gate_name="SystemdWatchdog" >}}
```

Comment:
It seems like there’s an issue with the PR build failing _[(click here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6715156eb41751000823824d)_ due to a missing feature gate description file for 'SystemdWatchdog'.
> 3:37:46 PM: ERROR Invalid feature gate: 'SystemdWatchdog' is not recognized or lacks a matching description file in 'en/docs/reference/command-line-tools-reference/feature-gates'

_Please refer to the documentation on how to add this here: [k8s.io/docs/contribute/new-content/new-features](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates)._

Reply:
Thank you for your review